mms2/java/java-datatypes/src/DataTypes_App.java

```diff
@@ -0,0 +1,129 @@
+import com.datastax.driver.core.Cluster;
+import com.datastax.driver.core.ResultSet;
+import com.datastax.driver.core.Row;
+import com.datastax.driver.core.Session;
+import com.datastax.driver.core.PreparedStatement;
+import com.datastax.driver.core.BoundStatement;
+import com.datastax.driver.core.*;
+import com.datastax.driver.core.utils.Bytes;
+import java.io.*;
+import java.nio.ByteBuffer;
+import java.nio.channels.FileChannel;
+import java.util.HashMap;
+import java.util.Map;
+
+public class DataTypes_App {
+
+static File FILE;
+
+public static void main(String[] args) {
+        try {
+                Cluster cluster = Cluster.builder().addContactPoints("scylla-node1", "scylla-node2", "scylla-node3").build();
+                Session session = cluster.connect("catalog");
+                createSchema(session);
+
+                String[] mutants = new String[] {"Jim Jefferies", "Bob Loblaw", "Bob Zemuda"};
+                for (int i=0; i != mutants.length; i++) {
+                        if(mutants[i] != null) {
+                                String[] first_name = mutants[i].split(" ");
+                                String last_name = first_name[1];
+                                FILE = new File("/opt/code/" + first_name[0] + "_" + last_name + ".png");
```

Comment:
`/opt/code/` should be `/opt/code/java-datatypes`

Reply:
Thanks, fixed and tested this